Task2/Task2/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace Task2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.Write("Введите натуральное число N: ");
+            string n = Console.ReadLine();
+            if (int.TryParse(n, out int N))
+            {
+                if (N >= 0)
+                {
+                    Console.WriteLine($"\nЧётные натуральные числа от 0 до {N}:");
+                    for (int i = 1; i <= N; i++)
+                    {
+                        if (i % 2 == 0)
+                        {
+                            Console.WriteLine($"{i}");
+                        }
+                    }
+                }
+                else Console.WriteLine("Натуральное число не может быть отрицательным.");
```

Comment:
```suggestion
                else 
                {
                     Console.WriteLine("Натуральное число не может быть отрицательным.");
                }
```

Always wrap conditions with brackets, even when single-line.